packages/turf-bezier/index.js

```diff
@@ -13,6 +13,7 @@ var Spline = require('./spline.js');
  * @param {number} [resolution=10000] time in milliseconds between points
  * @param {number} [sharpness=0.85] a measure of how curvy the path should be between splines
  * @returns {Feature<LineString>} curved line
+ * @addToMap line, curved
```

Comment:
This was where I was thinking the layer control box might be handy. On some maps it's going to make sense to show both the input and output (I think bezier is a good example where it's helpful to show both). It should be very simple to introduce